synthetic-monitoring/sm.libsonnet

```diff
@@ -0,0 +1,107 @@
+{
+  _new(name, target):: {
+    frequency: 60000,
+    offset: 0,
+    timeout: 2500,
+    enabled: true,
+    labels: [],
+    target: target,
+    job: name,
+  },
+
+  local _new = self._new,
+
+  http: {
+    new(name, target):: _new(name, target) + {
+      settings: {
+        http: {
+          ipVersion: 'V4',
+          method: 'GET',
+          noFollowRedirects: false,
+          failIfSSL: false,
+          failIfNotSSL: false,
+        },
+      },
+    },
+  },
+
+  tcp: {
+    new(name, target):: _new(name, target) + {
+      settings: {
+        tcp: {
+          ipVersion: 'V4',
+          tlsConfig: {},
```

Comment:
Empty object?

Reply:
that's what the API returns for a manually created check, so yes!